docs/tutorial/part03.rst

```diff
@@ -2,5 +2,245 @@
 How to create the base models.
 ==============================
 
-.. todo::
-   Here will be the tutorial body.
\ No newline at end of file
+Create a new app for the API: 
+==============================
+Right now, we need to create an app which is going to be use  to create the API and, with it, the models definition. If you do not know the definition of an API; it is “`code that allows two software programs to communicate with each other`”. 
+
+To create this new app, it is needed to run this command: 
+
+``python manage.py startapp fetch_api``
+
+Where ``fetch_api`` is the name of the application we are creating, if you prefer you can use any other name.  This will create a new folder inside the program we have, and the structure of it is: ::

```

Comment:
Instead of "inside the program we have", say "inside the Django project directory". Remember readers will know, or it will serve as a reminder, that a Django project can have as many apps as they want. We are just creating a Django app and we assume readers are familiar it.